experimenter/experimenter/experiments/constants.py

```diff
@@ -411,6 +411,8 @@ class EmailType(models.TextChoices):
     ERROR_FIREFOX_VERSION_MIN = (
         "Ensure this value is less than or equal to the maximum version"
     )
+    ERROR_FIREFOX_VERSION_MIN_96 = "The minimum targetable Firefox version is 96"
+    ERROR_FIREFOX_VERSION_MIN_REQUIRED = "Experiments must set a minimum Firefox version"
```

Comment:
Where is this used?

Reply:
`ERROR_FIREFOX_VERSION_MIN_REQUIRED` isnt used anymore.

Reply:
But why is it in the PR?

Reply:
I left it in by accident. A previous version had different messages when the version was too low vs not present at all. I replaced it with a single message so that people wouldnt hit both errors if they tried to set No version -> a version too low.

Reply:
Yep makes sense 👍